### ISSUE TITLE
Breaking error

### ISSUE THREAD

**Issue by night** [cletusc/Userscript--Twitch-Chat-Emotes#95]
Might be related to twitch prime. Also check out https://github.com/night/BetterTTV/issues/1574

```
Uncaught TypeError: Cannot read property 'toLowerCase' of null
(anonymous function) @ betterttv.js?0.2071224904488358:241
r.addGroup @ betterttv.js?0.2071224904488358:241
r.addEmote @ betterttv.js?0.2071224904488358:241
(anonymous function) @ betterttv.js?0.2071224904488358:241
r.updateEmotes @ betterttv.js?0.2071224904488358:241
i.updateEmotes @ betterttv.js?0.2071224904488358:241
(anonymous function) @ betterttv.js?0.2071224904488358:240
(anonymous function) @ betterttv.js?0.2071224904488358:241
```

**Comment by cletusc** [comment]
:(

**Comment by cletusc** [comment]
Fixed by e08b2fd26c3351f89c5e8c1d26bbea2c1d3bdab6
